lesson35/lesson35_part1.py

```diff
@@ -39,9 +37,6 @@ def purchase_opportunity(price_per_unit: int, amount: int):
         return f'To buy {amount} products, you need {final_price} hryvnias'
```

Comment:
Method returns string, incompatible with docstring.
In general, this string might be logging message, but message should return final_price, because code that will use this method shouldn't parse it to get value